.storybook/main.ts

```diff
@@ -10,7 +10,6 @@ const config: StorybookConfig = {
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
     'storybook-dark-mode',
-    '@chakra-ui/storybook-addon',
```

Comment:
it added no value and rendered a second dark mode button which didn't work fine, because we have chakra+rebass :(